src/lib/nodes/rate-info-service/RateInfoService.ts

```diff
@@ -31,11 +30,20 @@ export class RateInfoService extends Node<AxiosInstance> {
 
   protected async checkHealth(): Promise<HealthcheckResult> {
     const start = Date.now()
-    const response = await this.getAllRates()
-    this.height = normalizeHeight(response.date)
+    this.height = (await this.getAllRates()).date || 0;
```

Comment:
```suggestion
    const { date } = await this.getAllRates()
    this.height = date;
```

Much more readable in my opinition. Also the `date` is a required param, no need a fallback to `0`